src/region/RegionAnnotations.tsx

```diff
@@ -106,6 +115,7 @@ export default class RegionAnnotations extends React.Component<Props, State> {
                             key={id}
                             annotationId={id}
                             isActive={!isCreating && activeAnnotationId === id}
+                            onSelect={this.handleAnnotationSelect}
```

Comment:
Is there a difference between active/selected? We're using `isActive` and `activeAnnotation` above.

Reply:
I don't think there's any differentiation right now, would `handleAnnotationActive` be more appropriate?